packages/test/test-utils/src/loaderContainerTracker.ts

```diff
@@ -15,7 +15,7 @@ import {
 	IContainerCreateProps,
 	IContainerLoadProps,
 	// eslint-disable-next-line import/no-internal-modules
-} from "@fluidframework/container-loader/dist/container";
+} from "@fluidframework/container-loader/test/container";
```

Comment:
Needs to use path setup in 'container-loader' exports (instead of fishing around in /dist/)